src/cli/serve/serve.js

```diff
@@ -212,8 +212,7 @@ export default function (program) {
         },
         features: {
           isClusterModeSupported: CAN_CLUSTER,
-          isOssModeSupported: !IS_KIBANA_DISTRIBUTABLE,
-          isXPackInstalled: XPACK_INSTALLED,
+          stripXpackConfig: XPACK_INSTALLED && opts.oss,
```

Comment:
why `XPACK_INSTALLED` should affect this logic?

Reply:
I copied the logic that causes the x-pack plugin to be initialized, but inverted the oss flag which is only available to set when running from source anyway.

Reply:
Ah, good call